generators/app/templates/travis.yml

```diff
@@ -1,13 +1,14 @@
-sudo: false
 language: node_js
-cache:
-  directories:
-    - ~/.npm
+cache: npm
```

Comment:
Wait, does this do the same thing as I was trying to do?

Reply:
Like I explained above (https://github.com/kentcdodds/generator-kcd-oss/pull/13#discussion_r370940614), they're the same indeed 🙂